gensrc/proto/internal_service.proto

```diff
@@ -204,13 +204,15 @@ message PTabletWriterCancelResult {
 };
 
 message PExecPlanFragmentRequest {
+    required bool pad = 1;
```

Comment:
Use optional all the time.
And you should leave comments to explain why there is pad field.

Reply:
ok, i will add comments. because if the request is null the brpc will throw an exception.